website/docs/implementation_guides/maintainer/checklist/readme.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Checklist
+sidebar_label: Checklist
+---
+
+# Checklist
+
+## Hygiene
+
+* [ ] Project created from template
+* [ ] Documentation populated from template
+* [ ] Documentation synced via automation to docs.pact.io
+* [ ] Issue templates added
+* [ ] Labels added
+* [ ] Example projects created
+* [ ] Tutorials/workshops to get started
+* [ ] Supports minimum OS/Arch combinations?
+
+## Quality
+
+* [ ] TCK completed and published
```

Comment:
Just a thought for checklist, from previous threads

sustainable CI, or repo pre-reqs, good practises.

https://github.com/pact-foundation/devrel/issues/8

in addition, docs, ie

- contributing guide
- developer guide
- readme 

etc

Reply:
Yep, nice ideas. I just popped a few off the top of my head, but I think it will likely be getting down what we have initially with some "TODOs" that we can populate over time as needed.